apps/meteor/ee/server/apps/orchestrator.js

```diff
@@ -201,10 +206,97 @@ export class AppServerOrchestrator {
 		await Promise.all(apps.map((app) => this.getNotifier().appUpdated(app.getID())));
 	}
 
+	async findUpgradeToV7Date() {
+		let hadPre7Version = false;
+		let upgradeToV7Date = null;
+
+		try {
+			const statistics = await this.getStatisticsModel().findInstallationDates();
+			if (statistics && statistics.length > 0) {
+				const sortedStats = statistics.sort((a, b) => new Date(a.installedAt) - new Date(b.installedAt));
+
+				for (const stat of sortedStats) {
+					const version = stat.version || '';
+
+					if (version && !version.startsWith('7.')) {
```

Comment:
Wouldn't it be a problem when we start to have versions > 7? I mean, by doing this, we will need to revisit this in the future, is this expected?